includes/options/class-amp-options-manager.php

```diff
@@ -17,38 +17,19 @@ class AMP_Options_Manager {
 	 */
 	const OPTION_NAME = 'amp-options';
 
-	/**
-	 * Slug for website experience.
-	 */
-	const WEBSITE_EXPERIENCE = 'website';
-
-	/**
-	 * Slug for stories experience.
-	 *
-	 * @var
-	 */
-	const STORIES_EXPERIENCE = 'stories';
-
 	/**
 	 * Default option values.
 	 *
 	 * @var array
 	 */
 	protected static $defaults = [
-		'experiences'             => [ self::WEBSITE_EXPERIENCE ],
 		'theme_support'           => AMP_Theme_Support::READER_MODE_SLUG,
 		'supported_post_types'    => [ 'post' ],
 		'analytics'               => [],
 		'all_templates_supported' => true,
 		'supported_templates'     => [ 'is_singular' ],
 		'enable_response_caching' => true,
 		'version'                 => AMP__VERSION,
-		'story_templates_version' => false,
```

Comment:
Removed Story relation settings; see https://github.com/ampproject/amp-wp/pull/4315/commits/ceab0121f1d15c6c666153e02191f71da1e5b0ac.

Reply:
Shouldn't that be a database upgrade routine instead?

This right now causes database queries on every page load.

Reply:
Addressed in https://github.com/ampproject/amp-wp/pull/4315/commits/d67421e0c73ecccd20f9b3db9980f6edf3157286.